skate-plugin/build.gradle.kts

```diff
@@ -24,6 +24,7 @@ plugins {
   alias(libs.plugins.intellij)
   alias(libs.plugins.pluginUploader)
   alias(libs.plugins.buildConfig)
+  alias(libs.plugins.compose)
```

Comment:
We may also need the kotlin.multiplatform plugin here in place of kotlin.jvm but not sure

Reply:
I was thinking since Skate is Intellij plugin, having kotlin.jvm should be enough. Does having kotlin.multiplatform mean we'll extend to other platforms like iOS?

Reply:
Not necessarily, it just sets up the KMP structure. If you look at the build file in the original repo, it does the same thing.

Reply:
ah I see. I can try moving the KMP structure over